pdata/internal/cmd/pdatagen/internal/metrics_structs.go

```diff
@@ -489,6 +493,15 @@ var exemplar = &messageValueStruct{
 	},
 }
 
+var dataPointFlagsFieldStruct = &messageValueField{
+	fieldName:       "FlagsStruct",
```

Comment:
Open to suggestions on this name.

Reply:
Sounds reasonable.

Reply:
The existing `Flags()` api doesn't actually show as deprecated when used bc it auto-generated code. Is that ok or should we flag it and point people to `FlagsStruct`?

Reply:
I think we should mark that as deprecated, otherwise cannot expect people to change. We can do that in a separate PR.

Reply:
Ok.